pages/posts/postman-case-study.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "How Postman uses JSON Schema"
+date: 2023-07-17T06:00:00+00:00
+type: Case Study
+cover: /img/posts/2023/postman-case-study/cover.webp
+authors:
+  - name: Juan Cruz Viotti
+    photo: /img/avatars/jviotti.webp
+    link: https://twitter.com/jviottidc
```

Comment:
Looks like this account doesn't exist anymore (well done :)

Reply:
@jviotti Would you like to provide a new link? =]

Reply:
I opted out of Twitter after recent changes. You can use my website: https://www.jviotti.com